### DISCUSSION TITLE
Interested in sponsorship

### DISCUSSION THREAD

**Discussion by jeetraut** [ping-pub/explorer#298] [Q&A]
Please email me at jeet (at) agoric.com to follow-up.

Thanks

**Comment by liangping** [comment]
@jeetraut ,thanks for connecting, sent you an email and looking forward to talk more details.

**Comment by jeetraut** [comment]
Thanks, I’m reviewing with my team and I’ll get back to you after the
holidays.

Best,

Jeet

On Wed, Dec 21, 2022 at 2:02 AM ping ***@***.***> wrote:

> @jeetraut <https://github.com/jeetraut> ,thanks for connecting, sent you
> an email and looking forward to talk more details.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ping-pub/explorer/discussions/298#discussioncomment-4465047>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABSLF7N2EIR265MPNUVIG33WOKTQDANCNFSM6AAAAAATETT4WI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
-- 
Jeet Raut
347-755-0472
LinkedIn <http://linkedin.com/in/jeetraut/>
jeetraut.com

**Comment by liangping** [comment]
Hwy. However do you team think of this sponsorship?



---Original---
From: "Jeet ***@***.***&gt;
Date: Fri, Dec 23, 2022 00:15 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [ping-pub/explorer] Interested in sponsorship (Discussion #298)




 Thanks, I’m reviewing with my team and I’ll get back to you after the
 holidays.
 
 Best,
 
 Jeet
 
 On Wed, Dec 21, 2022 at 2:02 AM ping ***@***.***&gt; wrote:
 
 &gt; @jeetraut <https://github.com/jeetraut&gt; ,thanks for connecting, sent you
 &gt; an email and looking forward to talk more details.
 &gt;
 &gt; —
 &gt; Reply to this email directly, view it on GitHub
 &gt; <https://github.com/ping-pub/explorer/discussions/298#discussioncomment-4465047&gt;,
 &gt; or unsubscribe
 &gt; <https://github.com/notifications/unsubscribe-auth/ABSLF7N2EIR265MPNUVIG33WOKTQDANCNFSM6AAAAAATETT4WI&gt;
 &gt; .
 &gt; You are receiving this because you were mentioned.Message ID:
 &gt; ***@***.***&gt;
 &gt;
 -- 
 Jeet Raut
 347-755-0472
 LinkedIn <http://linkedin.com/in/jeetraut/&gt;
 jeetraut.com

**Reply by sybilsingleton** [reply]
Hello, sorry to bump the thread, but Dyson Protocol is interested in sponsoring Ping.

**Comment by liangping** [comment]
Hey. Sounds great. DM me on TG: pingpub
